dev-master/zen/cli.py

```
@@ -0,0 +1,170 @@
+        try:
+            return json.loads(content)
+        except json.JSONDecodeError:
+            try:
+                return yaml.safe_load(content) or {}
+            except yaml.YAMLError as exc:
+                raise click.ClickException(f"Failed to parse variables file: {exc}") from exc
+
+    try:
+        return json.loads(value)
+    except json.JSONDecodeError:
+        return parse_variables(value)
```

Ruff (0.14.10)

Undefined name `parse_variables`

(F821)